src/main/java/ca/sait/vezorla/controller/ClientRestController.java

```diff
@@ -43,6 +45,14 @@ public boolean updateAccount(@RequestBody Account account, HttpServletRequest re
         return created;
     }
 
+
+    @GetMapping("account/forgotpassword")

```

Comment:
`@PutMapping`

Reply:
Yes, we are altering the database. The change does work as we want to return a boolean to the frontend. I will make the change shortly.